cli/script/command-executor.ts

```diff
@@ -907,15 +936,34 @@ function printAccessKeys(format: string, keys: AccessKey[]): void {
     if (format === "json") {
         printJson(keys);
     } else if (format === "table") {
-        printTable(["Key", "Time Created", "Created From", "Description"], (dataSource: any[]): void => {
-            keys.forEach((key: AccessKey): void => {
-                dataSource.push([
-                    key.name,
+        printTable(["Name", "Time Created", "Created From", "Expires"], (dataSource: any[]): void => {
```

Comment:
I realize this isn't directly related to your PR, but should we rename "Time Created" to just "Created"? With the formatting that we use for the time strings, I think it reads more naturally.